auto/src/glew_head.c

```diff
@@ -6,7 +6,9 @@
 #  include <GL/glxew.h>
 #endif
 
+#include <alloca.h>
```

Comment:
I've been setting up a Win7 laptop for development, so I'll be able to check Windows, Linux and Mac.